src/api/app/views/webui/staging/workflows/edit.html.haml

```diff
@@ -12,6 +12,12 @@
               %i.fas.fa-plus-circle.text-success
               Create Staging Project
             = render partial: 'create_staging_project', locals: { staging_workflow: @staging_workflow }
+          %li.list-inline-item
```

Comment:
This view is behind authorization, so the `if policy(@staging_workflow).update?` from `staging_workflow#show` isn't needed anymore.